vllm_ascend/ops/vocab_parallel_embedding.py

```diff
@@ -72,3 +83,176 @@ def vocab_parallel_embedding_forward(self, input_):
 
 
 VocabParallelEmbedding.forward = vocab_parallel_embedding_forward
+
+
+class AscendVocabParallelEmbedding(VocabParallelEmbedding):
```

Comment:
can you rebase for this change? https://github.com/vllm-project/vllm-ascend/pull/2515/ does the same thing and it sovled quantization problem as well.